packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeDetailsItem/MT5TradeDetailsItem.tsx

```diff
@@ -17,39 +16,52 @@ type TMT5TradeDetailsItemProps = {
 const MT5TradeDetailsItem: FC<TMT5TradeDetailsItemProps> = ({ label, value, variant = 'clipboard' }) => {
     const { isDesktop } = useDevice();
     const { show } = useModal();
+
+    const textSize = isDesktop ? 'xs' : 'sm';
+
+    const TradeDetailsHeader = ({ children }: PropsWithChildren) => (
+        <Text color='less-prominent' size={textSize}>
+            {children}
+        </Text>
+    );
+
```

Comment:
hey @aizad-deriv 
although the current implementation shouldn't introduce any bugs,
but i would like to emphasize that creating new components inside parent components is anti-pattern that we should avoid, because it can cause to unpredictable consequences because we are losing component state on each parent's rerender.
I would like to ask extract this component outside the parent 🙏

Reply:
I ended up reverting back to the original. 👍🏼